### DISCUSSION TITLE
如何将洛雪音乐的歌单导入 MusicFree？

### DISCUSSION THREAD

**Discussion by SamLukeYes** [maotoumao/MusicFree#265] [Q&A]
我用洛雪音乐作为主力播放器已经一年多了。由于近期音源不稳定，并且了解到 MusicFree 的移动端功能更全面，想要迁移到 MusicFree。我的歌单虽然最早是来源于网易云的，但是由于已经在洛雪上维护了一年多，目前已经有许多变更，再加上 MusicFree 导入网易云歌单是乱序的，而且会过滤掉不少现在已经收费的曲目，先导入网易云歌单再对照洛雪的歌单手动添加少部分歌曲的方法并不可行。请问有什么别的迁移手段吗？

我注意到洛雪可以将歌单导出为 txt 或 csv 格式，能否利用其导出文件在 MusicFree 中进行导入呢？

**Comment by qgzng** [comment]
哥们疯了？两个软件不兼容的

**Comment by SamLukeYes** [comment]
已经重新养歌单了，现在我主要用本地曲库+m3u歌单。以高昂的时间成本绕过了这个问题。
